src/curp-command.proto

```diff
@@ -135,7 +136,7 @@ message FetchReadStateRequest {
 
 message FetchReadStateResponse {
     message IdSet {
-        repeated ProposeId ids = 1;
+        repeated uint64 inflight_ids = 1;
```

Comment:
Why should we use `uint64` to replace `ProposeId` here?

Reply:
`ProposeId` is a unique id composed of two u64 values. Here, it is hashed into a single u64 for better transmission and to encapsulate the concept of `ProposeId` within curp.
The new concept is called `InflightId` which is returned by curp to the client indicates that the command represented by this id is currently running. It notifies the client to cancel waiting for these ids during after-sync.